x-pack/legacy/plugins/grokdebugger/public/register.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { xpackInfo } from 'plugins/xpack_main/services/xpack_info';
+import { npSetup, npStart } from 'ui/new_platform';
+
+npSetup.plugins.devTools.register({
+  order: 6,
```

Comment:
Why 6? 😅

Reply:
I just copied it over from the old descriptor object 🙂
https://github.com/elastic/kibana/pull/49349/files#diff-9bb4d61e0628e7b18a885cbce72eebcaL13